src/main/kotlin/com/intershop/gradle/resourcelist/task/ResourceListFileTask.kt

```diff
@@ -39,6 +39,11 @@ abstract class ResourceListFileTask
     @Inject constructor(objectFactory: ObjectFactory,
                         private val fileSystemOps: FileSystemOperations) : DefaultTask() {
 
+    /**
+     * The one and only folder searched for resources
+     */
+    private val RESOURCES = "resources"
```

Comment:
This is java and this is Kotlin ...

companion object {
        private const val RESOURCES = "resources"
    }